### PR TITLE
Integratioin Guide for Data Ingestion from Upstash Kafka into RisingWave

### PR DESCRIPTION
**Pull request by fahadullahshah261**
This PR involves the integration guide for Upstash Kafka. Please review it for any potential improvements. Thanks.

### CODE REVIEW

File: docs/ingest/ingest-from-upstash-kafka.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+id: ingest-from-upstash-kafka
+title: Ingest data from Upstash Kafka
+description: Connect RisingWave to a Kafka broker in Upstash.
+slug: /ingest-from-upstash-kafka
+---
+<head>
+  <link rel="canonical" href="https://docs.risingwave.com/docs/current/ingest-from-upstash-kafka/" />
+</head>
+
+# Ingesting Data from Upstash Kafka to RisingWave
```
Comment (ShanlanLi) [inline docs/ingest/ingest-from-upstash-kafka.md]: We use sentence cases in the documentation, so maybe these headings should be uncapitalized.

File: docs/ingest/ingest-from-upstash-kafka.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+You can ingest data from Kafka deployed in Upstash into RisingWave. Upstash is a serverless platform that offers Redis, Kafka, and Qstash, providing the benefits of scalability, advanced security options, and dedicated support.
+
+## Set Up Kafka on Uptash
+
+This guide goes through the steps to create a Kafka cluster on Uptash and to connect it to RisingWave for data ingestion. For more information regarding the data ingestion from Upstash, please refer to [Upstash Documentaion](https://upstash.com/docs/kafka/overall/getstarted).
```
Comment (ShanlanLi) [inline docs/ingest/ingest-from-upstash-kafka.md]: We use sentence cases in references too. Maybe this should be uncapitalized too. Here are the detailed guidelines: https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings

File: docs/ingest/ingest-from-upstash-kafka.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+)
+WITH(
+connector = 'kafka',
+topic = '<topic-name>', 
+properties.bootstrap.server = '<broker-url>', 
+scan.startup.mode = 'earliest', 
+properties.sasl.mechanism = 'SCRAM-SHA-512', 
+properties.security.protocol = 'SASL_SSL', 
+properties.sasl.username = '<your-username>', 
+properties.sasl.password = '<your-password>'
```
Comment (neverchanje) [inline docs/ingest/ingest-from-upstash-kafka.md]: please add a 2 spaces tab before each of the field.

File: docs/ingest/ingest-from-upstash-kafka.md (hunk 1 above)
Comment (CharlieSYH) [reply docs/ingest/ingest-from-upstash-kafka.md]: Updated

**Review by CharlieSYH — APPROVED** [review]
Made a few adjustments.
Good to go. Thanks, @fahadullahshah261!

**Review by WanYixian — APPROVED** [review]
LGTM!

**Review by hengm3467 — APPROVED** [review]

**Pull request merged**